modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm

```diff
@@ -21,9 +21,23 @@ limitations under the License.
 
 Bio::EnsEMBL::Compara::PipeConfig::Lastz_conf
 
+=head1 SYNOPSIS
+
+    Standard pipeline initialisation:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Lastz_conf -host mysql-ens-compara-prod-X -port XXXX \
+            -division $COMPARA_DIV -mlss_id_list "[1596,1583,1570,1562]"
+
+    [Alternative 1] Provide the collection and the non reference species:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Lastz_conf -host mysql-ens-compara-prod-X -port XXXX \
+            -division $COMPARA_DIV -collection hagfish -non_ref_species eptatretus_burgeri
+
+    [Alternative 2] Provide the collection and the reference species:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Lastz_conf -host mysql-ens-compara-prod-X -port XXXX \
+            -division $COMPARA_DIV -collection collection-e94_new_species_human_lastz -ref_species homo_sapiens
+
 =head1 DESCRIPTION  
 
-This is a base configuration files for all LastZ runs.
+This is a base configuration file for LastZ pipeline.
 It is not intended to be run on its own, it just turns the generic
 PairAligner pipeline into a LastZ one.
```

Comment:
The instructions above make it seem like it's fine to run on its own, so maybe remove this?